api/app/Models/PoolCandidate.php

```diff
@@ -87,166 +87,109 @@ public function scopeClassifications(Builder $query, ?array $classifications): B
         return $query;
     }
 
-    public function filterByOperationalRequirements(Builder $query, ?array $operationalRequirements): Builder
+    public function scopeOperationalRequirements(Builder $query, ?array $operationalRequirements): Builder
     {
-        // if no filters provided then return query unchanged
-        if (empty($operationalRequirements)) {
+        if (empty($workRegions)) {
             return $query;
         }
 
-        // OperationalRequirements act as an AND filter. The query should only return candidates willing to accept ALL of the requirements.
-        $query->whereExists(function ($query) use ($operationalRequirements) {
-            $query->select('id')
-            ->from('users')
-            ->whereColumn('users.id', 'pool_candidates.user_id')
-            ->whereJsonContains('accepted_operational_requirements', $operationalRequirements);
+        // point at filter on User
+        $query->whereHas('user', function ($query) use ($workRegions) {
+            User::scopeLocationPreferences($query, $workRegions);
         });
 
         return $query;
```

Comment:
Copy paste error? (workRegions instead of operationRequirements)

Reply:
Thanks!